plugin-server/tests/shared/process-event.ts

```diff
@@ -136,7 +136,7 @@ export const createProcessEventTests = (
     async function processEvent(
         distinctId: string,
         ip: string | null,
-        siteUrl: string,
+        _siteUrl: string,
```

Comment:
This looks like an interesting flyby change?

Reply:
I was obeying the linter. Unused parameter